webpack-intro/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "webpack-intro",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "jquery": "^3.6.0"
+  },
+  "devDependencies": {
+    "cli": "^1.0.1",
```

Comment:
If you install a package by mistake, like this `cli` package, you can uninstall it with:
```
npm uninstall package-name
```

So for the `cli` package you'd do:
```
npm uninstall cli
```

Reply:
@scbowler Thank you! I wasn't sure so I just left it as is.